packages/plugin-stripe/src/admin.ts

```diff
@@ -1,20 +1,20 @@
 import type { Config } from 'payload/config'
 
-import type { SanitizedStripeConfig, StripeConfig } from './types.js'
+import type { PluginConfig, SanitizedPluginConfig } from './types.js'
 
 import { getFields } from './fields/getFields.js'
 
 export const stripePlugin =
-  (incomingStripeConfig: StripeConfig) =>
+  (incomingPluginConfig: PluginConfig) =>
   (config: Config): Config => {
     const { collections } = config
 
     // set config defaults here
-    const stripeConfig: SanitizedStripeConfig = {
-      ...incomingStripeConfig,
+    const pluginConfig: SanitizedPluginConfig = {
+      ...incomingPluginConfig,
       // TODO: in the next major version, default this to `false`
```

Comment:
We should get @jacobsfletch 's input on this comment here.

Reply:
Will address in subsequent PR if needed.

Reply:
Great catch 👍 I'll take care of this